pkg/test/environment.go

```diff
@@ -118,7 +119,7 @@ func NewEnvironment(ctx context.Context, env *coretest.Environment) *Environment
 	securityGroupProvider := securitygroup.NewDefaultProvider(ec2api, securityGroupCache)
 	versionProvider := version.NewDefaultProvider(env.KubernetesInterface, kubernetesVersionCache)
 	instanceProfileProvider := instanceprofile.NewDefaultProvider(fake.DefaultRegion, iamapi, instanceProfileCache)
-	ssmProvider := ssmp.NewDefaultProvider(ssmapi, ssmCache)
+	ssmProvider := ssmp.NewDefaultProvider(ssmapi, ssmCache, clock, "https://test-cluster")
```

Comment:
Do we have a variable for fake.ClusterEndpoint?

Reply:
We don't atm, right now we just inject `https://test-cluster` anywhere we pass through the ClusterEndpoint. We discussed this a bit offline, but long term I imagine expanding the `Version` provider into a `Cluster` provider, which would provide any information related to the host kubernetes cluster: k8s version, cluster endpoint, cluster cidr, etc. Then the fake cluster endpoint could just be driven through the fake EKS interface.